commands/help.js

```diff
@@ -13,15 +14,26 @@ exports.run = (client, message, args) => {
         continue;
       var commandName = fileName.replace('.js', '').replace('./', '');
       var commandMessage = `**${commandName}**: \`${usage}\` ${helpMessage}`;
-      var newLine = (index === (files.length - 1)) ? null : "\n";
-      reply += commandMessage + newLine;
+      var newLine = (index === (files.length - 1)) ? null : '\n';
+      if (replies[replyIndex]) {
+        replies[replyIndex] += (commandMessage + newLine);
+        if (replies[replyIndex].length > 1000)
+          replyIndex++;
+      } else 
+        replies[replyIndex] = (commandMessage + newLine);
     }
-    message.author.send(reply)
+    var promises = [];
+    for (var index = 0; index < replies.length; index++ ) {
+      var promise = message.author.send(replies[index]);
+      promises.push(promise);
+    }
+    Promise.all(promises)
+    .then()
     .catch(() => {
-      message.reply("You might want to enable DM's first :facepalm:");
+      message.reply('You may want to enable DM\'s first :facepalm:');
     });
   });
 };
 
-exports.usage = "+8ball [question]";
-exports.helpMessage = "Sends you a message with details on other commands you can use.";
+exports.usage = '+8ball [question]';
```

Comment:
Shouldn't `usage` just be `'+help'` and not `'+8ball [question]'`?

Reply:
Yes it should be cheers for this